modules/slugs/includes/qtranslate-slug-settings.php

```diff
@@ -171,7 +171,7 @@ function qts_show_form_field( $args = array() ) {
                     $checked = 'checked="checked"';
                 }
 
-                echo "<label for='$id|$item_value'><input class='radio$field_class' type='radio' id='$id|$item_value' name='" . QTX_OPTIONS_MODULE_SLUGS . "[$id]' value='$item_value' $checked /> <strong>$item_key</strong>";
+                echo "<label for='qtx_slugs_$id'><input class='radio$field_class' type='radio' id='qtx_slugs_$id' name='" . QTX_OPTIONS_MODULE_SLUGS . "[$id]' value='$item_value' $checked /> <strong>$item_key</strong>";
```

Comment:
I removed most of the HTML `id` attributes from the Slugs settings form because they created confusions.
They are not needed, unlike the `name` attributes that matter for the form POST.